examples/05_operationalize/als_movie_o16n.ipynb

```diff
@@ -112,7 +112,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I think we can remove this cell and the previous comment, in other spark notebooks we don't add it. Everything is explained in the SETUP. </p><p> </p><p>Maybe what happened is that someone was debuging the notebook and added <code>os.environ</code>  and forgot to remove before commiting. Then another person added the comment. Just a theory :-) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I'm thinking making changes in this PR.
For now the issue we discussed is, the notebook contains two separate contents 
1: using cosmosDB, 2: using AzureML for realtime prediction.

Two options are: a) Two separate sections in one notebook vs. b) Have two separate notebooks. Which one make more sense?

Reply:
I think having two separate notebooks provides more clarity

Reply:
@miguelgfierro If we make two separate notebooks, most of the contents will be redundant.
Instead, I made changes to put the on-demand prediction part as an appendix markdown contents at the end of the notebook. Please check the changes and let me know if that looks good. I also cleaned up the env-variable cells you caught.

Reply:
ok